.github/workflows/test_on_push.yaml

```diff
@@ -11,7 +11,7 @@ jobs:
       github.event_name == 'pull_request' && github.event.pull_request.head.repo.full_name != github.repository
     strategy:
       matrix:
-        tarantool-version: ["1.10", "2.2", "2.3", "2.4", "2.5", "2.6", "2.7"]
+        tarantool-version: ["1.10", "2.2", "2.3", "2.7", "2.8", "2.9"]
```

Comment:
2.9 was never released. There was a 'technical release' 2.9.0, which is equivalent to 2.8.1. All development goes into 2.10. Now there is one pre-release: 2.10.0-beta1. See https://github.com/tarantool/setup-tarantool/issues/19.

Reply:
2.9 removed